core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -364,28 +366,24 @@ var TopicEditorPage = function() {
 
   this.navigateToTopicEditorTab = async function() {
     var topicEditorTab = element(by.css('.protractor-test-edit-topic-tab'));
-    await waitFor.elementToBeClickable(
-      topicEditorTab, 'Topic editor tab taking too long to be clickable');
-    await topicEditorTab.click();
+    await action.click('Topic editor tab', topicEditorTab);
   };
 
   this.navigateToSubtopicWithIndex = async function(subtopicIndex) {
     var subtopic = await subtopics.get(subtopicIndex);
-    await subtopic.click();
+    await action.click('Subtopic', subtopic);
     await waitFor.pageToFullyLoad();
   };
 
   this.expectNumberOfStoriesToBe = async function(count) {
-    if (count) {
-      await waitFor.visibilityOf(
-        storyListTable, 'Story list table takes too long to appear.');
-    }
+    await waitFor.visibilityOf(
+      storyListItems, 'Story list items element taking too long to appear');
     expect(await storyListItems.count()).toEqual(count);
   };
 
   this.expectStoryTitleToBe = async function(title, index) {
     await waitFor.visibilityOf(
-      storyListTable, 'Story list table takes too long to appear.');
+      storyListItems, 'Story list items element taking too long to appear');
```

Comment:
Here and elsewhere, I don't think we can use a `ElementArrayFinder` (element.all(..) returns this) in any of the waitFors or action calls.

Why make this change btw?

Ditto below where storyListTable is replaced with storyListItems.

Reply:
Yes, the storyListTable needn't be replaced. Modified in this and similar instances elsewhere.